articles/azure/azs-how-create-vm-from-snapshot-ps.md

```diff
@@ -0,0 +1,194 @@
+---
+title: How to create a virtual machine from a disk snapshot using PowerShell
+description: Provides help for creating a virtual machine from a managed disk snapshot on UKCloud for Microsoft Azure using PowerShell
+services: azure-stack
+author: William Turner
+reviewer: William Turner
+lastreviewed: 06/05/2020
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a VM from a Disk Snapshot
+toc_sub3:
+toc_sub4:
+toc_title: Create a VM from a disk snapshot - PowerShell
+toc_fullpath: Users/How To/Create a VM from a Disk Snapshot/azs-how-create-vm-from-snapshot-ps.md
+toc_mdlink: azs-how-create-vm-from-snapshot-ps.md
+---
+
+# How to create a virtual machine from a disk snapshot using PowerShell
+
+## Overview
+
+This article shows you how to use PowerShell to take a snapshot of a managed disk that is attached to a virtual machine, then use the snapshot to create a new managed disk and attach it to a new virtual machine from the new disk.
```

Comment:
Include the same text here from the Portal version to describe why customers might want to do this?

Reply:
Updated in latest commit.